### ISSUE TITLE
Cannot duplicate part

### ISSUE THREAD

**Issue by method1020** [inventree/InvenTree#4602]
### Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find a similar issue

### Describe the bug*

Receiving /api/part/ - An error occured in the current transaction. You can't execute queries until the end of the 'atomic' block.

### Steps to Reproduce

1. Select a part in the database.
2. Select the duplicate option.
3. Fill in duplicate part information, click okay.
4. Part does not duplicate and error appears.

### Expected behaviour

Part to duplicate.

### Deployment Method

- [X] Docker
- [ ] Bare metal

### Version Information

# Version Information:
InvenTree-Version: 0.10.1
Django Version: 3.2.16
Commit Hash: c1f15ef
Commit Date: 2023-02-25
Database: postgresql
Debug-Mode: False
Deployed using Docker: True
Active plugins: [{'name': 'InvenTreeBarcode', 'slug': 'inventreebarcode', 'version': '2.0.0'}, {'name': 'InvenTreeCoreNotificationsPlugin', 'slug': 'inventreecorenotificationsplugin', 'version': '1.0.0'}]

### Relevant log output

```shell
Traceback (most recent call last):

File "/root/.local/lib/python3.9/site-packages/rest_framework/views.py", line 506, in dispatch
response = handler(request, *args, **kwargs)

File "/root/.local/lib/python3.9/site-packages/rest_framework/generics.py", line 242, in post
return self.create(request, *args, **kwargs)

File "/usr/local/lib/python3.9/contextlib.py", line 79, in inner
return func(*args, **kwds)

File "/home/inventree/InvenTree/part/api.py", line 1231, in create
part.save(**{'add_category_templates': copy_templates})

File "/home/inventree/InvenTree/part/models.py", line 483, in save
for template in category.get_parameter_templates():

File "/root/.local/lib/python3.9/site-packages/django/db/models/query.py", line 280, in __iter__
self._fetch_all()

File "/root/.local/lib/python3.9/site-packages/django/db/models/query.py", line 1324, in _fetch_all
self._result_cache = list(self._iterable_class(self))

File "/root/.local/lib/python3.9/site-packages/django/db/models/query.py", line 51, in __iter__
results = compiler.execute_sql(chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size)

File "/root/.local/lib/python3.9/site-packages/django/db/models/sql/compiler.py", line 1175, in execute_sql
cursor.execute(sql, params)

File "/root/.local/lib/python3.9/site-packages/django/db/backends/utils.py", line 66, in execute
return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)

File "/root/.local/lib/python3.9/site-packages/django/db/backends/utils.py", line 75, in _execute_with_wrappers
return executor(sql, params, many, context)

File "/root/.local/lib/python3.9/site-packages/django/db/backends/utils.py", line 78, in _execute
self.db.validate_no_broken_transaction()

File "/root/.local/lib/python3.9/site-packages/django/db/backends/base/base.py", line 447, in validate_no_broken_transaction
raise TransactionManagementError(

django.db.transaction.TransactionManagementError: An error occurred in the current transaction. You can't execute queries until the end of the 'atomic' block.
```

**Comment by SchrodingersGat** [comment]
@method1020 thanks for reporting this. This bug has already been fixed in the [upcoming stable release](https://github.com/inventree/InvenTree/milestone/36) - but cannot be easily back ported due to large differences in the API. The new release will be out in the next few weeks, if you can cope until then?

**Comment by method1020** [comment]
This was resolved in 0.11.0 release. Tested. Thanks!
